[IMP] l10n_ar: changes from AFIP to ARCA and loc template/guidelines (#18190)
odoo/documentation: content/applications/finance/fiscal_localizations/argentina.rst

Argentina
=========

.. |ARCA| replace:: :abbr:`ARCA (Agencia de Recaudación y Control Aduanero)`

.. seealso::
   - `Webinar - Localización de Argentina <https://youtu.be/S5xlANeXja0>`_.

>    - `eCommerce - Localización de Argentina <https://www.youtube.com/watch?v=5gUi2WWfRuI>`_.

Reviewer: ```suggestion
   - `Webinar - Localización de Argentina <https://youtu.be/S5xlANeXja0>`_
   - `eCommerce - Localización de Argentina <https://www.youtube.com/watch?v=5gUi2WWfRuI>`_
```

   * - :guilabel:`Argentina - Accounting`
     - `l10n_ar`
     - Default :ref:`fiscal localization package <fiscal_localizations/packages>`, which represents
       the minimal configuration to operate in Argentina under the :abbr:`ARCA (Agencia de

>        Recaudación y Control Aduanero)` regulations and guidelines.

Reviewer: ```suggestion
       the minimal configuration to operate in Argentina under the |ARCA| regulations and guidelines.
```

     - `l10n_ar_withholding`
     - Allows registering withholdings during the payment of an invoice.

.. note::
   The localization's core modules are installed automatically with the localization. The rest can
   be manually :doc:`installed </applications/general/apps_modules>`.

.. _localizations/argentina/loc-review:

Localization overview
=====================

The Argentinian localization package ensures compliance with Argentinian fiscal and accounting
regulations. It includes tools for managing taxes, fiscal positions, reporting, and a predefined
chart of accounts tailored to Argentina's standards.

The Argentinian localization package provides the following key features to ensure compliance with
local fiscal and accounting regulations:

- :ref:`Chart of accounts <localizations/argentina/chart-accounts>`: a predefined structure tailored
  to Argentinian accounting standards
- :ref:`Taxes <localizations/argentina/taxes>`: pre-configured tax rates, including standard VAT,
  zero-rated, and exempt options
- :doc:`../accounting/taxes/fiscal_positions`: automated tax adjustments based on customer or

>   supplier registration status

Reviewer: Don't know that it matters, but in the French localization I just reviewed for you, these points were in the opposite order ...

- :ref:`Reporting <localizations/argentina/reporting>`

.. _localizations/argentina/chart-accounts:

Chart of accounts
-----------------

> In Accounting, three distinct :doc:`charts of accounts

Reviewer: Do you mean, 'In Odoo Accounting, ...'?
Would it be better just to start with: 'Three distinct ...'?

And also, maybe?:

```suggestion
In Accounting, three distinct :doc:`chart of accounts
```

(since we're talking about types of package, so using it as a modifier)

<../accounting/get_started/chart_of_accounts>` packages are available when :ref:`configuring the

> Argentinian fiscal localization <fiscal_localizations/packages>`. Selection depends on the company’s

Reviewer: Suggestion: 

```suggestion
Argentinian fiscal localization <fiscal_localizations/packages>`. The choice of package depends on the company’s
```

|ARCA| responsibility type and reflects the different levels of fiscal complexity required:

- :guilabel:`Monotributista` (227 accounts): Generic Chart of Accounts Argentina Single Taxpayer /
  Basis
- :guilabel:`Exento` (290 accounts): Argentine Generic Chart of Accounts for Exempt Individuals
- :guilabel:`Responsable Inscripto` (298 Accounts): Argentine Generic Chart of Accounts for

>   Registered Accountants.

Reviewer: ```suggestion
- :guilabel:`Monotributista` (227 accounts): Generic Chart of Accounts Argentina Single Taxpayer /
  Basis
- :guilabel:`Exento` (290 accounts): Argentine Generic Chart of Accounts for Exempt Individuals
- :guilabel:`Responsable Inscripto` (298 accounts): Argentine Generic Chart of Accounts for
  Registered Accountants
```

Is there a reason to use title case for the explanations each time?

.. _localizations/argentina/taxes:

Taxes
-----

As part of the localization module, taxes are created automatically with their related financial

> account and configuration, e.g., 73 taxes for :guilabel:`Responsable Inscripto`.

Reviewer: Not clear what you mean here ... I'm sure it's correct, but if you could help me understand, I'd appreciate it.

Argentina has several tax types; the most common ones are:

- :guilabel:`VAT`: Regular VAT with various percentages.
- :guilabel:`Perception`: Advance payment of a tax that is applied to invoices.
- :guilabel:`Retention`: Advance payment of a tax that is applied to payments.

> Some Argentinian taxes are not commonly used for all companies, and those less common options are

Reviewer: ```suggestion
Some Argentinian taxes are not commonly used for all companies, and are therefore
```

labeled as inactive in Odoo by default. Before creating a new tax, be sure to check if that tax is

> not already included as inactive.

Reviewer: ```suggestion
labeled as inactive in Odoo by default. Before creating a new tax, check that the tax is not already included and labeled inactive.
```

.. _localizations/argentina/document-types:

Document types
--------------

In some Latin American countries, like Argentina, accounting transactions such as invoices and
vendor bills are classified by document types defined by the governmental fiscal authorities. In
Argentina, the `ARCA <https://www.arca.gob.ar/>`__ is the governmental fiscal authority that
defines such transactions.

The document type is an essential piece of information that must be clearly displayed in printed
reports, invoices, and journal entries that list account moves.

Each document type can have a unique sequence per journal where it is assigned. As part of the
localization, the document type includes the country where the document is applicable (this data
is created automatically when the localization module is installed).

The information required for the :guilabel:`Document Types` is included by default, so the user must

> not fill anything in the :guilabel:`Document Types` view.

Reviewer: 'Must not' reads a little differently than 'does not need to' ... and I'm thinking it's the latter that's more accurate? 'Must not" reads more like 'is not allowed to'.

But I'd maybe suggest to reword it to something like this:

```suggestion
The required information about :guilabel:`Document Types` is included by default, so no additional information needs to be entered in the :guilabel:`Document Types` view.
```

But I'm not sure I fully understand .... do you mean the required information for each document type? Also, from the screenshot, it doesn't look like they could fill in anything anyway ...

_Edited later to add: I see you used the wording 'no further configuration is necessary.' ... could that work better here too?_

.. note::

>    The list displayed is not exhaustive and does not include all available document types.

Reviewer: Do you mean the list in the Document Types view? Might they need other document types that are not listed?

- :guilabel:`Exportation Transaction`, a document type :guilabel:`E` must be used.

The documents included in the localization already have the correct letter associated with each
:guilabel:`Document Type`, and no further configuration is necessary.

.. _localizations/argentina/doc-types-invoices:

Use on invoices
~~~~~~~~~~~~~~~

The :guilabel:`Document Type` on each transaction will be determined by:

- The journal entry related to the invoice (if the journal uses documents).
- The conditions applied depend on the issuer and receiver types (e.g., the buyer's and vendor's
  fiscal regime types).

.. _localizations/argentina/journals:

Journals
--------

In the Argentinian localization, the :doc:`journal <../accounting/get_started/journals>` can have
a different approach depending on its usage and internal type. To configure journals, go to
:menuselection:`Accounting --> Configuration --> Journals`.

For sales and purchase journals, the :guilabel:`Use Documents` option can be activated, enabling a
list of :guilabel:`Document Types` to be linked to invoices and vendor bills. For more details on
invoices, please refer to section :ref:`2.3 document types

> <localizations/argentina/document-types>`.

Reviewer: I guess the 'please refer to section ...' is a remnant of how the doc used to be structured and labeled?

Sholud you be dropping that sentence completely and just be linking on 'Document Types' back up to the 'Document types' section?

If the sales or purchase journals do not have the :guilabel:`Use Documents` option activated, they
will not be able to generate fiscal invoices, limiting their use case mostly to monitoring account
moves related to internal control processes.

.. _localizations/argentina/journals-arca:

ARCA information (ARCA Point of Sale)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The :guilabel:`ARCA POS System` is a field that defines the type of ARCA POS to be used to manage
the transactions for which the journal is created.

The ARCA POS defines the following:

#. The sequences of document types related to the web service.

> #. The structure and data of the electronic invoice file.

Reviewer: Do these need to be numbered? I think they could just be regular bullets maybe

.. _localizations/argentina/journals-web-services:

Web services
************

**Web services** help generate invoices for different purposes. Below are a few options to choose
from in the :guilabel:`ARCA POS System` field:

- :guilabel:`wsfev1: Electronic Invoice`: Is the most common service, used to generate invoices for
  document types A, B, C, M, with no item detail.
- :guilabel:`wsbfev1: Electronic Fiscal Bond`: Is for those who invoice capital goods and wish to

>   access the benefit of the Electronic Tax Bonds granted by the Ministry of Economy. For more

Reviewer: Just 'benefit from the Electronic Tax Bonds', as later on the same page?

  details, go to: `Fiscal Bond
  <https://www.argentina.gob.ar/acceder-un-bono-por-fabricar-bienes-de-capital>`__.
- :guilabel:`wsfexv1: Export Voucher`: Is used to generate invoices for international customers and

>   transactions involving export processes; the document type related is type "E".

Reviewer: ```suggestion
- :guilabel:`wsfev1: Electronic Invoice`: The most common service, used to generate invoices for
  document types A, B, C, M, with no item detail.
- :guilabel:`wsbfev1: Electronic Fiscal Bond`: For those who invoice capital goods and wish to benefit from the `Electronic Tax Bonds <https://www.argentina.gob.ar/acceder-un-bono-por-fabricar-bienes-de-capital>`_ granted by the Ministry of Economy. 
- :guilabel:`wsfexv1: Export Voucher`: Used to generate invoices for international customers and
  transactions involving export processes; the related document type is type E.
```

Here are some useful fields to know when working with web services:

- :guilabel:`ARCA POS Number`: Is the number configured in |ARCA| to identify the operations
  related to this ARCA POS.

> - :guilabel:`ARCA POS Address`: Is the field related to the commercial address registered for the

Reviewer: ```suggestion
- :guilabel:`ARCA POS Number`: The number configured in |ARCA| to identify the operations
  related to this ARCA POS.
- :guilabel:`ARCA POS Address`: The field related to the commercial address registered for the
```

  POS, which is usually the same address as the company. For example, if a company has multiple
  stores (fiscal locations), then |ARCA| will require the company to have one ARCA POS per
  location. This location will appear on the invoice report.

In ARCA, when configuring the electronic invoicing data, the following should be configured:

- MONOTRIBUTISTA: Factura Electrónica – Monotributo - WebServices
- RESPONSABLE INSCRIPTO: RECE para aplicativo y WebServices

> - IVA EXENTO: Facturación Electrónica - exento en IVA - WebServices

Reviewer: Is there a reason for the all caps? In the 'Chart of accounts' section, they are just sentence case and bold:

<img width="627" height="331" alt="Image" src="https://github.com/user-attachments/assets/f2308fd9-25de-4961-b963-ee97ac1a4677" />

.. _localizations/argentina/journals-sequence:

Sequences
~~~~~~~~~

For the first invoice, Odoo automatically synchronizes with ARCA and displays the last sequence
used.

.. note::
   When creating :guilabel:`Purchase Journals`, specify if they are related to document types. When
   the :guilabel:`Use Documents` option is activated, there is no need to manually associate
   document type sequences, since the vendor provides the document number.

.. _localizations/argentina/reporting:

Reporting
---------

As part of the localization installation, financial :doc:`reporting <../accounting/reporting>` for
Argentina is available in the :guilabel:`Accounting` dashboard. To access these reports, navigate to
:menuselection:`Accounting --> Reporting`, in the :guilabel:`Argentinean Statements`

> section.

Reviewer: Maybe more like:

```suggestion
:menuselection:`Accounting --> Reporting`, and select the relevant report in the :guilabel:`Argentinean Statements` section.
```

To access the VAT book report, go to :menuselection:`Accounting --> Reporting --> Tax Report`, click
:icon:`fa-book` :guilabel:`Report:`, and select :guilabel:`VAT book (AR)`.

.. note::

>    Click the :icon:`fa-cog` menu and select :guilabel:`VAT Book (ZIP)` or :guilabel:`VAT Simple

Reviewer: ```suggestion
   Click the :icon:`fa-cog` :guilabel:`(cog)` icon and select :guilabel:`VAT Book (ZIP)` or :guilabel:`VAT Simple
```

   Report (ZIP)` to generate the VAT Book or VAT Simple report ZIP file, respectively.

> .. _localizations/argentina/vat-summary:

Reviewer: For these reports, maybe instead of 'is designed to' and 'allows you to validate', just:

'This pivot table displays ...[the monthly VAT totals] / [the gross income for each jurisdiction] / [the gross purchases for each jurisdiction]'.

VAT summary
~~~~~~~~~~~

This pivot table is designed to check the monthly VAT totals. This report is for internal use and is
not sent to the ARCA. To access it, go to :menuselection:`Accounting --> Reporting --> VAT Summary`.

.. _localizations/argentina/iibb-sales:

IIBB - Sales by jurisdiction
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

This pivot table allows you to validate the gross income for each jurisdiction. It serves as an
affidavit for the corresponding taxes due, but is not submitted to the ARCA. To access it, go to
:menuselection:`Accounting --> Reporting --> IIBB Sales by jurisdiction`.

.. _localizations/argentina/iibb-purchases:

IIBB - Purchases by jurisdiction
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

This pivot table allows you to validate the gross purchases for each jurisdiction. It serves as an
affidavit for the corresponding taxes due, but is not submitted to the ARCA. To access it, go to
:menuselection:`Accounting --> Reporting --> IIBB --> Purchases by jurisdiction`.

.. _localizations/argentina/company:

Company
=======

> Once the :ref:`localization modules are installed <localizations/argentina/modules>`, the first step

Reviewer: Do you think we really need to link to the modules?

is to set up the :ref:`company's record <general/companies/configuration>`. In addition to the basic

> information, make sure to complete the Ecuador-specific field, :guilabel:`ARCA Responsibility Type`,

Reviewer: Ecuador > Argentina?

to define the company's fiscal obligation and structure.

.. _localizations/argentina/contact:

Contacts
========

.. _localizations/argentina/contact-id-vat:

Identification type and VAT
---------------------------

As part of the Argentinian localization, document types defined by |ARCA| are available in the
:doc:`contact form <../../essentials/contacts>`. Information is essential for most transactions.

> Six :guilabel:`Identification Types` are available by default, as well as 32 inactive types.

Reviewer: In a 19.0 runbot with an Argentinian company, I'm not seeing any thing about document types only 'ARCA Responsibility Type':

<img width="1166" height="637" alt="Image" src="https://github.com/user-attachments/assets/7eb88740-3c34-45b8-9baf-e00474e42526" />

Also, in the 'ARCA responsibility type' section, you say that the document type is defined by the ARCA Responsibility Type ... so maybe the document types themselves are not / no longer mentioned on the contact form? 

What do you mean by 'Information is essential for most transactions.'?

And maybe the field name changed, but I'm seeing **Identification Number**, and 7 visible options:

<img width="493" height="287" alt="Image" src="https://github.com/user-attachments/assets/55ae9cef-d0ec-4735-a20e-cf3d9a9ecd61" />

So I think this needs to be updated.

.. note::
   The complete list of :guilabel:`Identification Types` defined by |ARCA| is included in Odoo,
   but only the common ones are active.

.. _localizations/argentina/contact-arca:

ARCA responsibility type
------------------------

In Argentina, the document type and corresponding transactions associated with customers and
vendors are defined by the :guilabel:`ARCA Responsibility Type`. This field should be defined in the
:doc:`contact form <../../essentials/contacts>`.

.. _localizations/argentina/e-invoicing:

E-invoicing
===========

.. _localizations/argentina/e-invoicing-config:

Configuration
-------------

.. _localizations/argentina/e-invoicing-environment:

Environment
~~~~~~~~~~~

The ARCA infrastructure is replicated across two separate environments, **testing** and
**production**.

Testing is provided so companies can test their databases until they are ready to move to the

> **Production** environment. As these two environments are completely isolated from each other, the

Reviewer: Any need for the bold and upper case on 'Production' here?

digital certificates issued in one instance are not valid in the other.

To select a database environment, go to :menuselection:`Accounting --> Configuration --> Settings`,
scroll down to the :guilabel:`Argentinean Localization` section, and choose either

> :guilabel:`Testing (Prueba)` or :guilabel:`Production (Produccion)`.

Reviewer: Maybe give a little more direction (also so we can do the same in the 'ARCA certificates' section next):

```suggestion
To select a database environment, go to :menuselection:`Accounting --> Configuration --> Settings`,
scroll down to the :guilabel:`Argentinean Localization` section. Then, under :guilabel:ÀRCA Web Services`, choose either :guilabel:`Testing (Prueba)` or :guilabel:`Production (Produccion)`.
```

.. _localizations/argentina/e-invoicing-ARCA:

ARCA certificates
~~~~~~~~~~~~~~~~~

The electronic invoice and other |ARCA| services use the :guilabel:`Web Services (WS)` provided by
|ARCA|.

To enable communication with |ARCA|, follow these steps to request a :guilabel:`Digital

> Certificate` if you don't already have one:

Reviewer: ```suggestion
Certificate` if you do not already have one:
```

> Certificate` if you don't already have one:

Reviewer: Should we maybe also explain the path to where this is? I know it's also in the same Accounting > Configuration > Argentinean Localization [did you ever open a bugfix to get that spelling fixed? 😉 ] > ARCA Web Services?

#. Generate Renewal Request: In the :guilabel:`ARCA Web Services` section, click
   :icon:`oi-arrow-right` :guilabel:`Generate Renewal Request` to generate a `.csr` (certificate
   signing request) file to use in the |ARCA| portal to request the certificate.
#. Obtain Certificate (ARCA): Access the |ARCA| portal and follow the instructions described in
   `this document <https://drive.google.com/file/d/17OKX2lNWd1bjUt3NxfqcCKBkBh-Xlpo-/view>`_ to get
   a certificate.
#. Certificate and Private Key (Odoo): To upload the certificate to Odoo, click the
   :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon next to the :guilabel:`Certificate` field
   and select the corresponding file. Then add the existing private key that was created after

>    clicking :guilabel:`Generate Certificate`.

Reviewer: Suggestion:

```suggestion
#. **Generate renewal request**: In the :guilabel:`ARCA Web Services` section, click
   :icon:`oi-arrow-right` :guilabel:`Generate Renewal Request` to generate a `.csr` (certificate
   signing request) file to use in the |ARCA| portal to request the certificate.
#. **Obtain certificate (ARCA)**: Access the |ARCA| portal and follow the instructions described in
   `this document <https://drive.google.com/file/d/17OKX2lNWd1bjUt3NxfqcCKBkBh-Xlpo-/view>`_ to get
   a certificate.
#. **Upload certificate and add private key (Odoo)**: To upload the certificate into Odoo, click the
   :icon:`oi-arrow-right` :guilabel:`(Internal link)` icon next to the :guilabel:`Certificate` field
   and select the corresponding file. Then add the existing private key that was created after
   clicking :guilabel:`Generate Certificate`.
```

What do you mean by '... after clicking Generate Certificate'? Is this something that happens after clicking the --> and selecting the file? Not 100% clear to me what happens after selecting the file ... And I guess at some point we might want to tell them to save these changes?

.. tip::
   - When a created certificate expires, click :guilabel:`Generate Renewal Request` to generate a
     new certificate.
   - In case you must configure the Homologation Certificate, please refer to the |ARCA| official
     documentation: `Homologation Certificate
     <http://www.arca.gob.ar/ws/documentacion/certificados.asp>`_. Furthermore, Odoo allows the user
     to test electronic invoicing locally without a Homologation Certificate. The following message

>      will be in the chatter when testing locally:

Reviewer: Suggestion:

```suggestion
   - If you need to configure the Homologation Certificate, please refer to the `|ARCA| official
     documentation <http://www.arca.gob.ar/ws/documentacion/certificados.asp>`_. It is possible to test electronic invoicing locally without a Homologation Certificate; the following message
     appears in the chatter when testing locally:
```

     .. image:: argentina/local-testing.png
        :alt: Invoice validated locally because it is in a testing environment without testing
              certificate/keys.

.. _localizations/argentina/accounting:

Accounting
==========

.. _localizations/argentina/invoice:

> Invoice

Reviewer: ```suggestion
Invoices
```

-------

The information below applies to invoice creation once the partners and journals are created and

> properly configured.

Reviewer: I'm thinking we shouldn't be using the term 'partner', no?

- :guilabel:`B2C transaction`, a document type :guilabel:`B` must be used.
- :guilabel:`Exportation Transaction`, a document type :guilabel:`E` must be used.

The documents included in the localization already have the correct letter associated with each
:guilabel:`Document Type`, and no further configuration is necessary.

.. _localizations/argentina/doc-types-invoices:

Use on invoices
~~~~~~~~~~~~~~~

The :guilabel:`Document Type` on each transaction will be determined by:

- The journal entry related to the invoice (if the journal uses documents).
- The conditions applied depend on the issuer and receiver types (e.g., the buyer's and vendor's
  fiscal regime types).

.. _localizations/argentina/journals:

Journals
--------

In the Argentinian localization, the :doc:`journal <../accounting/get_started/journals>` can have
a different approach depending on its usage and internal type. To configure journals, go to
:menuselection:`Accounting --> Configuration --> Journals`.

For sales and purchase journals, the :guilabel:`Use Documents` option can be activated, enabling a
list of :ref:`Document Types <localizations/argentina/document-types>` to be linked to invoices and

> vendor bills.

Reviewer: ```suggestion
For sales and purchase journals, the :guilabel:`Use Documents` option can be activated, enabling :ref:`Document Types <localizations/argentina/document-types>` to be linked to invoices and
vendor bills.
```

The "... enabling a list of Document Types ..." sounds odd ... I guess it's just that an individual document type (from the list of available types) can be linked, is that right?

- :guilabel:`wsbfev1: Electronic Fiscal Bond`: For those who invoice capital goods and wish to
  benefit from the `Electronic Tax Bonds <https://www.argentina.gob.ar/acceder-un-bono-por-fabricar-bienes-de-capital>`_
  granted by the Ministry of Economy.
- :guilabel:`wsfexv1: Export Voucher`: Used to generate invoices for international customers and
  transactions involving export processes; the related document type is type E.

Here are some useful fields to know when working with web services:

- :guilabel:`ARCA POS Number`: The number configured in |ARCA| to identify the operations related to
  this ARCA POS.
- :guilabel:`ARCA POS Address`: The field related to the commercial address registered for the POS,
  which is usually the same address as the company. For example, if a company has multiple stores
  (fiscal locations), then |ARCA| will require the company to have one ARCA POS per location. This
  location will appear on the invoice report.

In ARCA, when configuring the electronic invoicing data, the following should be configured:

- Monotributista: Factura Electrónica – Monotributo - WebServices
- Responsable inscripto: RECE para aplicativo y WebServices

> - IVA exento: Facturación Electrónica - exento en IVA - WebServices

Reviewer: In the 'Chart of accounts' section, these are in the following order (and written list this):

Monotribusta
Extento
Responsable Inscripto 

Better to harmonise order, wording, and capitalisation?

#. Go to :menuselection:`Accounting --> Configuration --> Settings` and scroll down to the
   :guilabel:`Argentinean Localization` section.
#. **Generate renewal request**: In the :guilabel:`ARCA Web Services` section, click
   :icon:`oi-arrow-right` :guilabel:`Generate Renewal Request` to generate a `.csr` (certificate
   signing request) file to use in the |ARCA| portal to request the certificate.
#. **Obtain certificate (ARCA)**: Access the |ARCA| portal and follow the instructions described in
   `this document <https://drive.google.com/file/d/17OKX2lNWd1bjUt3NxfqcCKBkBh-Xlpo-/view>`_ to get
   a certificate.
#. **Upload certificate and add private key (Odoo)**: To upload the certificate into Odoo, click the
   :icon:`oi-arrow-right` :guilabel:`(Internal link)` icon next to the :guilabel:`Certificate` field
   and select the corresponding file. Then add the existing private key that was created after
   clicking :guilabel:`Generate Renewal Request`.

.. tip::
   - When a created certificate expires, click :guilabel:`Generate Renewal Request` to generate a
     new certificate.
   - If you need to configure the Homologation Certificate, please refer to the `|ARCA| official

>      documentation <http://www.arca.gob.ar/ws/documentacion/certificados.asp>`_. It is possible to

Reviewer: ```suggestion
   - If you need to configure the Homologation Certificate, refer to the `official ARCA
     documentation <http://www.arca.gob.ar/ws/documentacion/certificados.asp>`_. It is possible to
```

Even though some invoices use the same journal, the prefix and sequence are given by the
:guilabel:`Document Type` field.

The most common :guilabel:`Document Type` will be automatically defined for different combinations
of :ref:`ARCA responsibility type <localizations/argentina/contact-arca>`, but it can be updated

> manually by the user before confirming the invoice.

Reviewer: ```suggestion
of :ref:`ARCA responsibility type <localizations/argentina/contact-arca>`, but it can be updated
manually before confirming the invoice.
```

.. _localizations/argentina/e-invoice-elements:

Electronic invoice elements
~~~~~~~~~~~~~~~~~~~~~~~~~~~

When using electronic invoices, if all information is correct, the invoice is posted in the standard
way unless an error needs to be addressed. Error messages indicate both the issue that needs
attention and a proposed solution. If an error persists, the invoice remains in draft until the
issue is resolved.

> Once the invoice is posted, the information related to |ARCA| result appears on the

Reviewer: ```suggestion
Once the invoice is posted, the information related to |ARCA| appears on the
```

invoice, and the :guilabel:`Result` field indicates if the invoice has been :guilabel:`Aceptado en
ARCA` or :guilabel:`Aceptado con Observaciones`.

In the :guilabel:`ARCA` tab, the following authorization information is displayed:

> - :guilabel:`ARCA authorization`: CAE number

Reviewer: CAE is mentioned three times on the page ... worth adding the abbr markup at the start of the page?

.. |CAE| replace:: :abbr:`CAE (Código de Autorización Electrónico)

Same question for FCE, which is mentioned a lot.

- :guilabel:`Authorization Due date`: deadline to deliver the invoice to the customers (normally 10
  days after the CAE is generated)

.. image:: argentina/arca-status.png
   :alt: ARCA Status.

.. _localizations/argentina/invoice-taxes:

Invoice taxes
~~~~~~~~~~~~~

> Based on the :guilabel:`ARCA Responsibility type`, the VAT tax can apply differently on the PDF

Reviewer: ```suggestion
Based on the :guilabel:`ARCA Responsibility Type`, the VAT tax can apply differently on the PDF
```

report:

- :guilabel:`A. Tax excluded`: In this case the taxed amount must be clearly identified in the
  report. This condition applies when the customer has the following :guilabel:`ARCA Responsibility

>   type` of :guilabel:`Responsable Inscripto`.

Reviewer: ```suggestion
- :guilabel:`A. Tax excluded`: In this case, the taxed amount must be clearly identified in the
  report. This condition applies when the customer's :guilabel:`ARCA Responsibility type` is :guilabel:`Responsable Inscripto`.
```

Also, what are the 'A' and 'B' mentioned at the start of these bullets?

- :guilabel:`B. Tax amount included`: This means that the taxed amount is included as part of the
  product price, subtotal, and totals. This condition applies when the customer has the following
  :guilabel:`ARCA Responsibility type`:

  - :guilabel:`IVA Sujeto Exento`
  - :guilabel:`Consumidor Final`
  - :guilabel:`Responsable Monotributo`
  - :guilabel:`IVA liberado`

.. _localizations/argentina/invoices-special-cases:

Special use cases
~~~~~~~~~~~~~~~~~

.. _localizations/argentina/invoices-services:

Invoices for services
*********************

For electronic invoices that include :guilabel:`Services`, |ARCA| requires reporting the service

> starting and ending dates. To do so, fill in the :guilabel:`Service Date` field in the

Reviewer: ```suggestion
start and end dates. To do so, fill in the :guilabel:`Service Date` field in the
```

:guilabel:`Other Info` tab.

.. note::
   If the dates are not selected manually before the invoice is validated, the values will be
   automatically set to the first and last days of the invoice's month.

.. _localizations/argentina/exportation-invoices:

Exportation invoices
********************

Invoices related to :guilabel:`Exportation Transactions` require a journal using |ARCA| POS
System :guilabel:`Export Voucher - Web Service` to ensure the correct document types are associated.

When the customer selected in the invoice is configured with an |ARCA| responsibility type
:guilabel:`Cliente / Proveedor del Exterior` - :guilabel:`Ley N° 19.640`, Odoo automatically
assigns the:

> - :guilabel:`Journal`: related to the exportation Web Service

Reviewer: Why the ':'?

- Exportation :guilabel:`Document Type`
- Exempt Taxes

In the :guilabel:`Other Info` tab:

- :guilabel:`Invoice` section - :guilabel:`ARCA Concept`: Products / Definitive export of goods
- :guilabel:`Accounting` section - :guilabel:`Fiscal position`: Compras/Ventas al exterior

.. note::

>    For Exportation Documents, complete the :guilabel:`Incoterm` field in the :guilabel:`Accounting`

Reviewer: Do we need the capitalization?

```suggestion
   For exportation documents, complete the :guilabel:`Incoterm` field in the :guilabel:`Accounting`
```

   section of the :guilabel:`Other Info` tab.

.. _localizations/argentina/fiscal-bond:

Fiscal bond
***********

The :guilabel:`Electronic Fiscal Bond` is used by those who invoice capital goods and wish to
benefit from the Electronic Tax Bonds granted by the Ministry of Economy.

For these transactions, it is important to consider the following requirements:

- Currency (according to the parameter table) and invoice quotation
- Taxes
- Zone
- Detail each item

  - Code according to the Common Nomenclator of Mercosur (NCM)
  - Complete description
  - Unit Net Price
  - Quantity
  - Unit of measurement
  - Bonus
  - VAT rate

.. _localizations/argentina/electronic-credit-invoice:

Electronic credit invoice MiPyme (FCE)
**************************************

For SME invoices, several document types are classified as **MiPyME**, also known as **Electronic
Credit Invoice** (**FCE** in Spanish). This classification develops a mechanism that improves
financing conditions for small and medium-sized businesses and allows them to increase productivity
by collecting early on credits and receivables issued to their clients and/or vendors.

> For these transactions, consider the following requirements:

Reviewer: Just above, you kept the '..., it is important to ...', so maybe keep it here too?

- Specific document types (201, 202, 206, etc).

> - The emitter should be eligible for the |ARCA| to MiPyME transactions.

Reviewer: Do we mean something like:

```suggestion
- The emitter should be eligible for MiPyME transactions via  |ARCA|.
```

Well, I guess it's about certifying their status as a MiPyME: https://www.argentina.gob.ar/produccion/registrar-una-pyme/certificado-pyme ... so maybe you can still find a better way of putting it.

- The amount should be larger than 100,000 ARS.

> - A bank account type CBU must be associated with the emisor; otherwise, the invoice cannot be

Reviewer: ```suggestion
- A bank account type CBU must be associated with the emitter; otherwise, the invoice cannot be
```

>   validated, with an error message such as the following.

Reviewer: ```suggestion
  validated, with an error message such as the following:
```

     transmission mode selected in :guilabel:`Settings`.

When creating a :guilabel:`Credit/Debit` note related to an FCE document:

- Use the :guilabel:`Credit Note` and :guilabel:`Debit Note` buttons in the related invoice form to
  transfer all information from the invoice to the new credit or debit note.
- The document letter should match the originator document (either A or B).
- The same currency as the source document must be used. When using a secondary currency, there is
  an exchange difference if the currency rate differs between the emission date and the payment
  date. Create a credit/debit note to decrease/increase the amount to pay in ARS.

When creating a credit note, there are two scenarios:

#. The FCE is rejected; then, the credit note should have the :guilabel:`FCE: is Cancellation?`
   option enabled, or
#. The credit note is created to annul the FCE document; then, the :guilabel:`FCE: is
   Cancellation?` option must be disabled.

.. _localizations/argentina/invoice-printed-report:

Invoice printed report
~~~~~~~~~~~~~~~~~~~~~~

The :guilabel:`PDF Report` for electronic invoices validated by |ARCA| includes a barcode at the

> bottom of the format, which represents the CAE number. The expiration date is also displayed, as it

Reviewer: '... of the format'? Do we mean at the bottom of the page? Also, it looks like a QR code rather than a barcode, no?

is a legal requirement.

.. image:: argentina/invoice-printed-report.png
   :alt: Invoice printed report.

.. _localizations/argentina/troubleshooting:

Troubleshooting and auditing
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

For auditing and troubleshooting purposes, detailed information can be obtained for an invoice
number previously sent to |ARCA|. To retrieve this information, follow these steps:

#. Activate the :ref:`developer mode <developer-mode>`.
#. Go to :menuselection:`Accounting --> Accounting --> Consult Invoice in ARCA`.
#. In the :guilabel:`Consult invoice in ARCA` window, select the relevant :guilabel:`Journal`,
   :guilabel:`Document Type`, and :guilabel:`Number`. Then, click :guilabel:`Get Invoice Detail`.

.. image:: argentina/consult-invoice-in-arca-details.png
   :alt: Details of invoice consulted in ARCA.

.. tip::
   To retrieve the last number used in ARCA for a specific :guilabel:`Document Type` and POS

>    :guilabel:`Number` and use this reference for any potential issues on the sequence

Reviewer: ```suggestion
   :guilabel:`Number` and use this reference for any potential issues with the sequence
```

   synchronization between Odoo and ARCA, select :guilabel:`Get Last Invoice` as the
   :guilabel:`Type`.

.. _localizations/argentina/vendor-bills:

Vendor bills
------------

Based on the selected purchase journal for the vendor bill, the :guilabel:`Document Type` field is
now required. This value is auto-populated based on the :ref:`ARCA Responsibility Type

> <localizations/argentina/contact-arca>` of Issuer and Customer, but it can be changed if necessary.

Reviewer: What do you mean '... of Issuer and Customer, ...'?
Issuer as in emitter of the invoice? But why the capitalization?

The :guilabel:`Document Number` field must be entered manually, and the format will be validated
automatically. However, if the format is invalid, a user error will be displayed indicating the

> expected format.

Reviewer: ```suggestion
automatically. However, if the format is invalid, an error message indicates the expected format.
```

The vendor bill number is structured the same way as customer invoices, except that the document
sequence is entered by the user in the following format: `Document Prefix - Letter - Document
Number`.

.. _localizations/argentina/validate-vendor-bill:

Validate vendor bill number in ARCA
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

As most companies have internal controls to verify that the vendor bill is related to a valid |ARCA|
document, an automatic validation can be set in :menuselection:`Accounting --> Configuration -->
Settings`. Scroll down to the :guilabel:`Argentinean Localization` section and consider the

> following levels in the :guilabel:`Verify Vendor Bills validity in ARCA` section:

Reviewer: ```suggestion
As most companies have internal controls to verify that the vendor bill is related to a valid |ARCA|
document, an automatic validation can be set. 

To do so, go to :menuselection:`Accounting --> Configuration -->
Settings`, scroll down to the :guilabel:`Argentinean Localization` section, then set one of the following options under :guilabel:`Verify Vendor Bills validity in ARCA`:
```

- :guilabel:`Not available:` the verification is not complete (this is the default value).
- :guilabel:`Available:` the verification is complete. If the number is not valid, it only displays
  a warning but still allows the vendor bill to be posted.
- :guilabel:`Required:` the verification is complete, and it does not allow the user to post the

>   vendor bill if the document number is not valid.

Reviewer: I think the edits are misconstruing this, because by 'done', they seem to mean 'performed'. So I think it's more:

```suggestion
- :guilabel:`Not available:` no verification is performed (this is the default value).
- :guilabel:`Available:` verification is performed. If the number is invalid, a warning is displayed but the vendor bill can still be posted.
- :guilabel:`Required:` verification is performed. The vendor bill can only be posted if the number is valid.
```

.. _localizations/argentina/validate-bill-odoo:

> Validate vendor bills in Odoo

Reviewer: Maybe too big an edit for now, but ... I'm not sure it makes sense to have 'Validate vendor bill number in ARCA'  and then 'Validate vendor bills in Odoo' as a sub-section. I think you just need a section 'Vendor bill number validatiion', where you first explain the settings and then explain the result (on vendor bills) of enabling those settings.

Settings`. Scroll down to the :guilabel:`Argentinean Localization` section and consider the
following levels in the :guilabel:`Verify Vendor Bills validity in ARCA` section:

- :guilabel:`Not available:` the verification is not done (this is the default value);
- :guilabel:`Available:` the verification is done. In case the number is not valid, it only displays
  a warning but still allows the vendor bill to be posted;
- :guilabel:`Required:` the verification is done, and it does not allow the user to post the vendor
  bill if the document number is not valid.
- :guilabel:`Not available:` the verification is not complete (this is the default value).
- :guilabel:`Available:` the verification is complete. If the number is not valid, it only displays
  a warning but still allows the vendor bill to be posted.
- :guilabel:`Required:` the verification is complete, and it does not allow the user to post the
  vendor bill if the document number is not valid.

.. image:: argentina/verify-vendor-bills.png
   :align: center
   :alt: Verify Vendor Bills validity in AFIP.
.. _localizations/argentina/validate-bill-odoo:

Validate vendor bills in Odoo
*****************************

> With the vendor validation settings enabled, a new button shows up on the vendor bills inside of

Reviewer: An important point to mention that this is something that only appears when the vendor bill number validation is enabled? Other wise, not clear where you're clicking 'Verify on ARCA'.

> Once the vendor validation settings are enabled, click :guilabel:`Verify on ARCA` in the

Reviewer: ```suggestion
Once the vendor bill validation settings are enabled, click :guilabel:`Verify on ARCA` in the
```

.. _localizations/argentina/fiscal-bond:

Fiscal bond
***********

The :guilabel:`Electronic Fiscal Bond` is used by those who invoice capital goods and wish to
benefit from the `Electronic Tax Bonds <https://www.argentina.gob.ar/acceder-un-bono-por-fabricar-bienes-de-capital>`_
granted by the Ministry of Economy.

For these transactions, it is important to consider the following requirements:

- Currency (according to the parameter table) and invoice quotation
- Taxes
- Zone
- Detail each item

  - Code according to the Common Nomenclator of Mercosur (NCM)
  - Complete description
  - Unit Net Price
  - Quantity
  - Unit of measurement
  - Bonus
  - VAT rate

.. _localizations/argentina/electronic-credit-invoice:

Electronic credit invoice MiPyme (FCE)
**************************************

For SME invoices, several document types are classified as **MiPyME**, also known as **Electronic
Credit Invoice** (|FCE| in Spanish). This classification develops a mechanism that improves
financing conditions for small and medium-sized businesses and allows them to increase productivity
by collecting early on credits and receivables issued to their clients and/or vendors.

> For these transactions, it's important to consider the following requirements:

Reviewer: ```suggestion
For these transactions, it is important to consider the following requirements:
```

- Specific document types (201, 202, 206, etc).
- The emitter should be eligible for MiPyME transactions via |ARCA|.
- The amount should be larger than 100,000 ARS.
- A bank account type CBU must be associated with the emitter; otherwise, the invoice cannot be
  validated, with an error message such as the following:

.. image:: argentina/bank-account-relation-error.png
   :alt: Bank account relation error.

To set up the :guilabel:`Transmission Mode`, go to :menuselection:`Accounting --> Configuration -->
Settings` and scroll down to the :guilabel:`Argentinean Localization` section. Then, in the

> :guilabel:`Default MiPyME |FCE| Transmission Option` section, select either:

Reviewer: ```suggestion
:guilabel:`Default MiPyME FCE Transmission Option` section, select either:
```

Because I guess the abbr functionality doesn't work when you have other rst formatting, like the guilabel in this case.

   owed to a supplier or a reduction in the total payment to be collected from a customer.
   Therefore, one or multiple withholdings can be recorded for each payment applied to an invoice.

.. _localizations/argentina/withholdings-configuration:

Configuration
~~~~~~~~~~~~~

> While Odoo already creates most of the required withholdings the :guilabel:`Taxes` menu, in several

Reviewer: Missing a word here I guess ...

Earnings
********

For :guilabel:`Earnings` withholdings, Odoo already has a record for each regime group, which is
listed under the tax name and the |ARCA| code.

Each of these records is ready to be used. As a best practice, the configuration should be
double-checked to make sure it is updated and well-applied. The fields to validate are:

- :guilabel:`Amount`: This is the percentage of the total payment amount which is withheld.
- :guilabel:`Non-Taxable Amount`: Up to this amount, the withholding does not apply.
- :guilabel:`Minimum Withholding`: If the calculated withholding amount is smaller than this value,
  the total withholding amount is set to `0.0`.
- :guilabel:`Withholding Sequence`: This field automates the capture of a withholding number under
  the payment line. If this field is not set, a number is manually captured while adding a
  withholding to a payment.

.. image:: argentina/l10n-ar-earnings.png
   :alt: Earnings withholding type.

.. _localizations/argentina/earnings-scale-withholdings:

Earnings Scale
**************

In this particular case, a percentage must not be set. Instead, this withholding is calculated based

> on the value of the :guilabel:`Scale` field.

Reviewer: Using 'must not be set' like this reads like it's forbidden to set a percentage.

I think the original is fine, and expresses the situation better.

To view, modify, or create new scales, navigate to :menuselection:`Accounting --> Configuration -->
Earnings Scale`. By default, the Argentinian localization is preconfigured with two main scales.
However, scales should be created and updated as necessary to suit a business's needs.

.. note::
   Earnings scales are cumulative, Odoo keeps track of the different records created for a bill and

>    automatically calculates the correct withholding amount.

Reviewer: To replace 'which means that', I think you need at least, 'i.e.'. Or, leave it as it was. But right now, the sentence isn't complete.

.. _localizations/argentina/iib-total-amount-withholdings:

IIBB Total Amount
*****************

In this case, the necessary records related to the applicable province must be created. The
withholding amount is calculated based on the percentage :guilabel:`Amount` set on the tax
configuration. Since Odoo does not automatically synchronize the percentages applicable to each
province, this information must be manually updated.

> The recommendation, in this case, is always to duplicate and apply the different configurations to

Reviewer: ```suggestion
The recommendation, in this case, is to always duplicate and apply the different configurations to
```

I'm struggling to explain why, but the original is better here

   :alt: Payment with applied withholdings.

.. important::
   The amount of the debt to be canceled equals the total payment amount. However, Odoo
   still captures the net amount (i.e., the amount to be reconciled with the bank), which will be
   represented as the payment amount after the withholding application.

   .. image:: argentina/l10n-ar-payment-registered.png
      :alt: Payment registered form.

.. _localizations/argentina/check-management:

Check management
----------------

Make sure to :ref:`install <general/install>` the :guilabel:`Third Party and Deferred/Electronic

> Checks Management` (`l10n_latam_check`) module.

Reviewer: As a note? (Like at the start of the Withholding management section.)

   :alt: Payment with applied withholdings.

.. important::

>    The amount of the debt to be canceled equals the total payment amount. However, Odoo

Reviewer: ```suggestion
   The amount of the debt to be cancelled equals the total payment amount. However, Odoo
```

The other two instances of 'cancelled' on this page (including one that's an actual field name) are spelled with two 'l's .... just sayin' 😉

Payments` tab.

- :guilabel:`Checks` should be available as a :guilabel:`Payment Method`. If not, click
  :guilabel:`Add a line` and type `Checks` under :guilabel:`Payment Method` to add them.
- Enable the :guilabel:`Use electronic and deferred checks` setting.

.. note::
   This last configuration **disables** the printing ability but enables to:

   - Enter check numbers manually.

>    - Adds a field to allocate the payment date of the check.

Reviewer: Sorry, I know you only added full stops, but the text needs some editing for it to be (more) correct and flow:

```suggestion
This last configuration disables the printing ability but:
- allows check numbers to be entered manually
- adds a field to indicate the payment date of the check
```

.. _localizations/argentina/own-checks-management:

Management of own checks
************************

Own checks can be created directly from the vendor bill. For this process, click :guilabel:`Register
Payment`.

> In the payment registration modal, select the bank journal from which the payment is to be made and

Reviewer: We don't use 'modal' in our doc, right?
I guess this is some kind of 'window'? And maybe a window entitled 'Payment Registration'?

.. note::
   To manage current checks, the :guilabel:`Check Cash-In Date` field must be left blank or filled
   in with the current date. To manage deferred checks, the :guilabel:`Check Cash-In Date` must be
   set in the future.

To manage your existing own checks, navigate to :menuselection:`Accounting --> Vendors --> Own
Checks`. This window displays critical information, including the dates checks must be paid, the
total number of checks, and the total amount paid in checks.

It is important to note that the list is pre-filtered by checks that are still *not reconciled* with
a bank statement - that were not yet debited from the bank - which can be verified with the
:guilabel:`Is Matched with a Bank Statement` field. If you want to see all of your own checks,
delete the :guilabel:`No Bank Matching` filter by clicking on the :guilabel:`X` symbol.

.. _localizations/argentina/own-check-cancel:

Cancel an own check
*******************

To cancel an own check created in Odoo, navigate to :menuselection:`Accounting --> Vendors --> Own
Checks` and select the check to be cancelled, then click :guilabel:`Void Check`. This breaks the

> reconciliation between vendor bills and bank statements, leaving the check marked as **cancelled**.

Reviewer: Original text mentioned 'a cancelled state' ... and you've written 'marked as cancelled'. If there's some kind of 'mark' or label, then use a guilabel?

.. _localizations/argentina/third-party-checks:

Third-party checks
~~~~~~~~~~~~~~~~~~

To register payments with third-party checks, two specific journals must be configured. To do so,
navigate to :menuselection:`Accounting --> Configuration --> Journals` and create two new journals:

- `Third-Party Checks`
- `Rejected Third-Party Checks`

.. note::
   You can manually create additional journals if you have multiple points of sale and need journals
   for those.

To create the *Third-Party Checks* journal, click :guilabel:`New` and configure the following:

- Type `Third-Party Checks` as the :guilabel:`Journal Name`.
- Select :guilabel:`Cash` as :guilabel:`Type`.
- In the :guilabel:`Journal Entries` tab, set :guilabel:`Cash Account`: to `1.1.1.02.010 Cheques de
  Terceros`, input a :guilabel:`Short Code` of your choice, and select a :guilabel:`Currency`.

> The available payment methods are listed in the *payments* tab:

Reviewer: If that's the tab name:

```suggestion
The available payment methods are listed in the :guilabel:`Payments` tab:
```

time of collection or upon receipt from vendors.

To create the *Rejected Third-Party Checks* journal, click :guilabel:`New` and configure the
following:

- Type `Rejected Third Party Checks` as the :guilabel:`Journal Name`
- Select :guilabel:`Cash` as :guilabel:`Type`
- In the :guilabel:`Journal Entries` tab, set :guilabel:`Cash Account`: to `1.1.1.01.002 Rejected
  Third Party Checks`, input a :guilabel:`Short Code` of your choice, and select a
  :guilabel:`Currency`.

Use the same payment methods as the *Third-Party Checks* journal.

.. _localizations/argentina/new-third-party-checks:

New third-party checks
**********************

To register a *new* third-party check for a customer invoice, click :guilabel:`Register Payment`. In
the pop-up window, select :guilabel:`Third-Party Checks` as the journal for the payment
registration.

Select :guilabel:`New Third Party Checks` as :guilabel:`Payment Method`, and fill in the
:guilabel:`Check Number`, :guilabel:`Payment Date`, and :guilabel:`Check Bank`. Optionally, you can
manually add the :guilabel:`Check Issuer Vat`, but it is automatically filled with the customer's

> VAT number on the invoice.

Reviewer: This was just a bit tricky to read ... not sure my suggestion is better:

```suggestion
manually add the :guilabel:`Check Issuer Vat`, but this field is automatically filled with the customer's
VAT number as it appears on the invoice.
```

electronic document for the liquidity product.

To modify the existing purchase journal or create a new one, navigate to :menuselection:`Accounting
--> Configuration --> Journals`. Then, select the existing purchase journal or click
:guilabel:`New`, and fill in the following required information:

- :guilabel:`Type`: Select :guilabel:`Purchase`.
- :guilabel:`Use Documents`: Check this field to select the electronic document type.
- :guilabel:`Is ARCA POS?`: Check this field to generate electronic documents. Three additional
  fields appear:

  - :guilabel:`ARCA POS System`: Select :guilabel:`Electronic Invoice - Web Service` from the
    drop-down menu to send the electronic document to |ARCA| via web service.
  - :guilabel:`ARCA POS Number`: Is the number configured in |ARCA| to identify the operations
    related to this ARCA POS.
  - :guilabel:`ARCA POS Address`: Is the field related to the commercial address registered for the
    POS, which is usually the same address as the company. For example, if a company has multiple
    stores (fiscal locations) then |ARCA| will require the company to have one ARCA POS per

>     location. This location will be printed in the invoice report.

Reviewer: ```suggestion
  - :guilabel:`ARCA POS Number`: The number configured in |ARCA| to identify the operations
    related to this ARCA POS.
  - :guilabel:`ARCA POS Address`: The field related to the commercial address registered for the
    POS, which is usually the same address as the company. For example, if a company has multiple
    stores (fiscal locations) then |ARCA| will require the company to have one ARCA POS per
    location. This location will be printed in the invoice report.
```

then sell the same product. This journal will not be synced with |ARCA| as the invoice will not be
electronic.

To modify the existing sales journal or create a new one, navigate to
:menuselection:`Accounting --> Configuration --> Journals`. Then, access the sales journal or click
:guilabel:`New`, and fill in the following required information:

- :guilabel:`Type`: select :guilabel:`Sales`.
- :guilabel:`Use Documents`: check this field on the journal to select the electronic document type
  (in this case, the electronic invoice).

.. _localizations/argentina/liquidity-invoicing-flow:

Invoicing flow
~~~~~~~~~~~~~~

Once all configurations are set, the *Liquidity Product Vendor Bill* will be generated by the
company selling the product on behalf of another party. For example, a distributor of a specific
product.

.. _localizations/argentina/ecommerce-electronic-invoicing:

Ecommerce electronic invoicing
==============================

:ref:`Install <general/install>` the :guilabel:`Argentinean eCommerce` (`l10n_ar_website_sale`)
module to enable the following features and configurations:

> - Clients being able to create online accounts for eCommerce purposes.

Reviewer: ```suggestion
- Allow clients to create online accounts for eCommerce purposes.
```
